internal/exec/terraform.go

```diff
@@ -245,10 +245,23 @@ func ExecuteTerraform(info schema.ConfigAndStacksInfo) error {
 		info.ComponentEnvList = append(info.ComponentEnvList, fmt.Sprintf("TF_APPEND_USER_AGENT=%s", appendUserAgent))
 	}
 
+	// Check for existing var-file arguments in TF_CLI environment variables
+	tfCommands := []string{"plan", "apply", "refresh", "import", "destroy", "console"}
+	for _, cmd := range tfCommands {
+		envVar := fmt.Sprintf("TF_CLI_ARGS_%s", cmd)
+		existing := os.Getenv(envVar)
+		if existing != "" && strings.Contains(existing, "-var-file=") {
```

Comment:
replace this behavior. Check for any "TF_" prefixed env var and give a warning. Do not emit the value, but do emit the name of the variable